src/components/pages/GenerationPage/PetitionList.js

```diff
@@ -79,7 +131,43 @@ function PetitionRow({ attorney, petition, petitionerData, validateInput }) {
             ))}
           </Attachments>
         </TableCell>
+        <TableCell>
+          <GenerateButton 
+            label={isDetailed ? <FontAwesomeIcon icon={faChevronDown}/> : <FontAwesomeIcon icon={faChevronRight}/>}
+            isCollapsed=<FontAwesomeIcon icon={faChevronDown}/>
+            onClick={() => handlePress()}
+            title="Reveal offense records"
+          />
+        </TableCell>
       </TableRow>
+      {isDetailed && (
+        <TableRow key={petition.pk + 'offenses'} backgroundColor={backgroundColor}>
```

Comment:
You don't need `key` prop unless you're using something like `list.map(() => <div key={key} />`

https://reactjs.org/docs/lists-and-keys.html